inference-engine/src/transformations/src/transformations/common_optimizations/add_fake_quantize_fusion.cpp

```diff
@@ -21,7 +22,17 @@ ngraph::pass::AddFakeQuantizeFusion::AddFakeQuantizeFusion() {
     auto input_pattern = ngraph::pattern::any_input();
     auto const_pattern = ngraph::pattern::wrap_type<opset5::Constant>();
     auto add_pattern = ngraph::pattern::wrap_type<opset5::Add>({input_pattern, const_pattern},
-                                                               pattern::consumers_count(1));
+            [] (const Output<Node>& node) -> bool {
+                if (node.get_target_inputs().size() != 1)
```

Comment:
If Constant has equal values or scalar value it can be fused into FQ in any case. @eshoguli am I right?